.github/workflows/continuous_integration.yml

```
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
```

```suggestion
        python-version: ["3.5", "3.6", "3.7", "3.8"]
```

We'll need to include this version for `lint` to run in `tox`.

3.5 is pretty close to EOL and I'd be fine with dropping (set up #71 to track it), but I recognize that this is module has a larger user base than some of our other projects. What's your thoughts?

---

I think it makes sense to keep 3.5 support.